synop2bufr/__init__.py

```diff
@@ -32,7 +32,7 @@
 from csv2bufr import BUFRMessage
 from pymetdecoder import synop
 
-__version__ = '0.4.1'
+__version__ = '0.5.0'
```

Comment:
Can you revert this to dev before we merge. We'll update the version number in a stand alone release.